package.json

```diff
@@ -235,13 +235,14 @@
     "is-elevated": "^3.0.0",
     "is-root": "^2.1.0",
     "js-yaml": "^3.14.0",
+    "JSONStream": "^1.0.3",
     "klaw": "^3.0.0",
     "livepush": "^3.5.0",
     "lodash": "^4.17.20",
     "minimatch": "^3.0.4",
     "mixpanel": "^0.10.3",
     "moment": "^2.27.0",
-    "moment-duration-format": "^2.3.2",
+    "moment-duration-format": "^2.3.2","net-keepalive": "^1.3.6",
```

Comment:
The build is failing on Windows because `net-keepalive` was added here. Was it added by accident?

Reply:
I guess so! I'll remove and commit